acceptance-tests/src/acceptance-test/java/tech/pegasys/teku/test/acceptance/validatorslashing/MultiPeersStandAloneVcBlocksAcceptanceTest.java

```diff
@@ -54,7 +54,7 @@ public class MultiPeersStandAloneVcBlocksAcceptanceTest
 
     firstTekuNode.start();
 
-    firstTekuNode.waitForEpochAtOrAbove(2);
+    firstTekuNode.waitForEpochAtOrAbove(1);
```

Comment:
if we're changing this anyway, i'd change this back to 2, and i'd change the genesis wait time to 60 seconds to let both nodes be online close to genesis...

Reply:
but i'd also be starting node 2 prior to waiting for epoch 2 i think...

Reply:
This wouldn't work because we would have to wait until epoch 2 to start the second node in the current logic. This would only drag the test length to a longer period of waiting. What we could do is possibly change the slashing to be in a block for the epoch 4 or 5.

Reply:
the BN will come up slow, so if the second BN starts before genesis and BOTH BN wait for the 2 epochs, then the second VC starts, by then the world order is a lot less ugly i think you'll find.